zookeeper-server/src/main/java/org/apache/zookeeper/server/ZKDatabase.java

```diff
@@ -304,6 +304,7 @@ public long fastForwardDataBase() throws IOException {
     }
 
     private void addCommittedProposal(TxnHeader hdr, Record txn, TxnDigest digest) {
+        LOG.debug("Creating a committed proposal request with null cnxn");
```

Comment:
Should we also print a `toString()` version of the `Request` object being created here?